packages/gitbook/src/components/DocumentView/Block.tsx

```diff
@@ -104,6 +106,12 @@ export function Block<T extends DocumentBlock>(props: BlockProps<T>) {
                 return <IntegrationBlock {...props} {...contextProps} block={block} />;
             case 'synced-block':
                 return <BlockSyncedBlock {...props} {...contextProps} block={block} />;
+            case 'reusable-content':
+                return null;
+            case 'stepper':
+                return <Stepper {...props} {...contextProps} block={block} />;
+            case 'stepper-step':
+                return <StepperStep {...props} {...contextProps} block={block} />;
```

Comment:
You might not need this one, see how it's doen for tabs, but you can render directly the children in Stepper

Reply:
Both are fine